pkg/node/getdatastoreentries.go

```diff
@@ -64,3 +64,48 @@ func DatastoreEntry(client *Client, address string, key string) (*DatastoreEntry
 
 	return &entry[0], nil
 }
+
+type GetDatastoreEntriesString struct {
+	Address string `json:"address"`
+	Key     string `json:"key"`
+}
+
+func DatastoreEntries(client *Client, params []GetDatastoreEntriesString) (*[]DatastoreEntryResponse, error) {
```

Comment:
Does it make sens to remove DatastoreEntry or simply to change it content to DatastoreEntries[0] ?